### DISCUSSION TITLE
Restoring settings back to default from installer file

### DISCUSSION THREAD

**Discussion by Khaztaroth** [valinet/ExplorerPatcher#1516] [Ideas]
After installing the program I set it to keep the windows 11 taskbar, not have the "properties" menu on right-click, thought I set it to be on the "win+x" menu. It seems that last setting doesn't apply to the win11 style menu which means now I have no way to access the preferences for ExplorerPatcher. Uninstalling doesn't remove the settings and I can't find a way to manually remove saved configurations. Removing files in the %appdata% folder doesn't change anything, neither does removing the registry keys. It would be useful to have a "load default config" option from the installer similar to how we can uninstall the program it from it. That way we can revert back to stock if some setting goes wrong. 

I couldn't find any indication that this is a feature already, apologies if there is already a way to achieve this.

**Comment by pyrates999** [comment]
In your start menu applications, look for **properties (explorer patcher)**.

**Reply by Khaztaroth** [reply]
What a lifesaver, for some reason I didn't think to look for anything there. Thanks

**Comment by ghost** [comment]
> In your start menu applications, look for **properties (explorer patcher)**.
> 

How can i revert to og settings in Windows 10? Please help

**Comment by pyrates999** [comment]
In **properties (explorer patcher)** application, go to about and click on Restore default settings:

![image](https://github.com/valinet/ExplorerPatcher/assets/106135298/281408ce-62bd-4faa-b8c9-ff6124b66e12)
